zk_toolbox/crates/zk_supervisor/src/commands/test/all.rs

```diff
@@ -0,0 +1,95 @@
+use std::thread;
+
+use common::{cmd::Cmd, config::global_config, logger};
+use xshell::{cmd, Shell};
+
+use super::{
+    args::{
+        all::AllArgs, integration::IntegrationArgs, recovery::RecoveryArgs, revert::RevertArgs,
+        upgrade::UpgradeArgs,
+    },
+    integration, recovery, revert, upgrade,
+};
+
+pub fn run(shell: &Shell, args: AllArgs) -> anyhow::Result<()> {
+    logger::info("Run integration tests");
```

Comment:
should we move this text into `integration::run` i think it's the same for all attempts to run this test

Reply:
please do

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/2594/commits/fc93244dd5ae0682488eb6ec078e193c7f56231f